packages/@textlint/kernel/src/task/fixer-task.ts

```diff
@@ -66,8 +65,9 @@ export default class TextLintCoreTask extends CoreTask {
         // setup "filters" field
         debug("filterRules", this.filterRuleDescriptors);
         this.filterRuleDescriptors.descriptors.forEach(filterRuleDescriptor => {
-            const ruleContext = createFreezedFilterRuleContext({
+            const ruleContext = new TextlintFilterRuleContext({
                 ruleId: filterRuleDescriptor.id,
+                severityLevel: getSeverity(filterRuleDescriptor.normalizedOptions),
```

Comment:
`TextlintFilterRuleContext`(aka. `RuleContext`) receive only `severityLevel`.
Currently, rule option is not needed.